scripts/infra

```diff
@@ -71,6 +71,17 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                 ;;
             apply)
                 terraform apply "${PFB_SETTINGS_BUCKET}.tfplan"
+
+                popd
+                pushd "${DIR}/.."
+
+                aws s3 cp "s3://${PFB_SETTINGS_BUCKET}/tilegarden/.env" "./src/tilegarden/.env"
+                aws s3 cp "s3://${PFB_SETTINGS_BUCKET}/tilegarden/claudia.json" "./src/tilegarden/claudia.json"
```

Comment:
Claudia creates this `claudia.json` file the first time it deploys a new function. After running that first deployment, I copied the `claudia.json` file into the S3 config bucket so we can use it as remote state.